app/editor/package.json

```diff
@@ -11,6 +11,7 @@
     "yarn": ">=3.2.0 <4.0.0"
   },
   "dependencies": {
+    "@bcgov/bc-sans": "^1.0.1",
```

Comment:
For the app we don't use a range of versions, as this will result in unpredictable differences between each environment.

Reply:
Updated, thanks